polygon/rest/aggs.py

```diff
@@ -78,15 +81,19 @@ def get_grouped_daily_aggs(
             result_key="results",
             deserializer=GroupedDailyAgg.from_dict,
             raw=raw,
+            options=options,
         )
 
+    # TODO: next breaking change release move "market_type" to be 2nd mandatory
+    # param
```

Comment:
I don't think this comment is related to the function below.

Reply:
this was added due to a line move.

Reply:
Im not sure why it's showing as an addition without a deduction. I didn't add this line. I can rollback if needed.